### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by Fokko**
Dependabot helps to keep the dependencies up to date.

### CODE REVIEW

File: .github /dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#
```
Comment (electrum) [inline .github /dependabot.yml:1]: Looks like there is a space in the file path. Also, we don't need the license header

File: .github /dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# specific language governing permissions and limitations
+# under the License.
+#
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (electrum) [inline .github /dependabot.yml:25]: Why not daily?

File: .github /dependabot.yml (hunk 1 above)
Comment (Fokko) [reply .github /dependabot.yml]: Great catch!

**Comment by electrum** [comment]
Thanks!

**Pull request merged**